src/richie/apps/courses/templates/courses/plugins/organization_small.html

```diff
@@ -0,0 +1,10 @@
+{% load cms_tags i18n %}
+{% spaceless %}
+{% if current_page.publisher_is_draft or instance.check_publication is True %}
+  {% with organization_page=relevant_page.organization.extended_object %}
+    <a class="organization-glimpse__small organization-glimpse organization-glimpse--link{% if organization_page.publisher_is_draft is True %} organization-glimpse--draft{% endif %}" href="{{ organization_page.get_absolute_url }}">
```

Comment:
The link is part of the template (it's inside the "component boundary") and should not be taken out. I previously fixed this because we had links sitting outside the component boundary.

If you want to apply a different class on some use sites it should be done through a prop you pass to the glimpse template

Reply:
@2018D good catch, and we already do it for the category here: https://github.com/openfun/richie/blob/master/src/richie/apps/courses/templates/courses/plugins/category_plugin.html

You should use the same pattern and pass a "form_factor" prop.